### PR TITLE
Refactor CSS

### PR DESCRIPTION
**Pull request by epicfaace**
- refactor a bunch of CSS
- upgrade react-scripts to allow using SASS

### CODE REVIEW

File: docker/compose_files/docker-compose.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ services:
 
   frontend:
     image: codalab/frontend:${CODALAB_VERSION}
-    command: serve -s build -l ${CODALAB_FRONTEND_PORT}
+    command: npm run build --production && serve -s build -l ${CODALAB_FRONTEND_PORT}
```
Comment (percyliang) [inline docker/compose_files/docker-compose.yml]: Why this change?  The files should already be built in the Docker image for production, and in dev mode, we hotload don't want to build the files anyway.

File: frontend/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "@material-ui/icons": "^3.0.2",
     "ace-builds": "^1.4.2",
     "autosuggest-highlight": "^3.1.1",
-    "bootstrap": "^4.3.1",
+    "bootstrap": "^3.3.6",
```
Comment (percyliang) [inline frontend/package.json:15]: Why are we going back in version?

**Review by percyliang — APPROVED** [review]
Thanks for doing this!  Everything looks good to me except the docker change (which probably should be a different PR anyway).

**Comment by percyliang** [comment]
@epicfaace can we merge this soon?

File: frontend/package.json (hunk 2 above)
Comment (epicfaace) [reply frontend/package.json:15]: We didn't use the `bootstrap` node module at all earlier; instead, we had just copied-and-pasted the bootstrap v3 styles into the imports.css file.

File: docker/compose_files/docker-compose.yml (hunk 1 above)
Comment (epicfaace) [reply docker/compose_files/docker-compose.yml]: Let me move that to another PR. The problem was that the way it worked before, whenever you started up a codalab instance in dev mode, you would have to wait for the files to build in production mode, which takes a long time.
